scripts/linters/warranted_angular_security_bypasses.py

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tuple of warranted angular safety bypasses in the codebase."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # isort:skip pylint: disable=import-only-modules
+
+# Contains the tuples of all files that bypasses angular's security mechanisms.
+# Please keep the list in alphabetical order.
+# NOTE TO DEVELOPERS: do not add any new files to the tuples without asking
+#  @seanlip first. You will have to explain the usage in the pr and the pr may
```

Comment:
This is a great note, but please check your spelling: 

pr --> PR

because

SECURITY

Reply:
Done!